app/ChartNotifier.js

```diff
@@ -0,0 +1,22 @@
+
+
+export default class ChartNotifier {
+    messageCallback = null;
+    removeByCategoryCallback=null
+
+    notify(_action, _data) {
+        if (_action === 'message') {
+            this.messageCallback(_data);
+        } else if (_action === 'removeByCategory') {
```

Comment:
just call `removeByCategory`. Why do you need `notify('removeByCategory', _data)` to call `removeByCategory`?

Reply:
I use `notify` for both notifying message and remove messages by category, Should I use separate these two methods?  `notify('removeByCategory', _data)` I use `_data` to pass category.

Reply:
@shayanbinary  yes it should be 2 methods. Think about it. What does verb `notify` have to do with removing notifications?

Reply:
@shayanbinary also. README documentation for `onMessage`?